RUFAS/data_validator.py

```
@@ -1870,6 +1871,70 @@ def validate_beef_cow_calf_config(config: dict[str, Any]) -> None:
         DataValidator._validate_beef_weaning_weight(config)
 
+    @staticmethod
+    def _validate_stocker_weight_order(config: dict[str, Any]) -> None:
+        """Raise ValueError when exit_weight does not exceed entry_weight."""
+        if (
+            "entry_weight" in config
+            and "exit_weight" in config
+            and config["entry_weight"] is not None
+            and config["exit_weight"] is not None
+        ):
+            if float(config["exit_weight"]) <= float(config["entry_weight"]):
```

Ruff (0.15.21)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)